src/components/visualizations/chart/highcharts/plugins/autohideLabels/autohideColumnLabels.ts

```diff
@@ -176,7 +220,11 @@ function toggleStackedLabelsForDualAxis() {
         const neighbors = toNeighbors(labels);
         const areOverlapping = areNeighborsOverlapping(neighbors);
 
-        if (areOverlapping) {
+        const areOverlappingColumns = areOverlapping
```

Comment:
I think we could have the naming more suggestive, and then use simple if condition with the two composed, something like (not tested): 
```
        const neighborsOverlapping = areNeighborsOverlapping(neighbors);
        const areOverlapping = neighborsOverlapping
            ? true
            : areLabelsOverlappingColumns(labels, getDataPointsOfVisibleSeries(this));

        if (areOverlapping) {
```

What would you think?

Reply:
updated in this commit https://github.com/gooddata/gooddata-react-components/pull/1106/commits/cf7d8cb98d0a0f19c1b165612e16891c480289d6